tests/unit/ReportUtilsTest.ts

```diff
@@ -294,6 +294,94 @@ describe('ReportUtils', () => {
         });
     });
 
+    describe('Automatically approved report message via automatic (not by a human) action is', () => {
+        test('Automatically Approved Report message when report is forwarded (Control feature)', () => {
```

Comment:
I think we can clean up these names a bit - I believe ideally it reads almost like a sentence, starting with the `describe` text, followed by the `test` itself - you can see how in other places (like just above) we have `describe('ParentReportAction is')` + `test('Manually Submitted Report Action')`